<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientEditDialog</class>
 <widget class="QDialog" name="clientEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QWidget" name="companyWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="cmpNameLabel">
        <property name="text">
         <string>Nazwa:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="cmpNameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cmpNIPLabel">
        <property name="text">
         <string>NIP:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cmpNIPLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cmpAddressLabel">
        <property name="text">
         <string>Adres:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="cmpAddressLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="prvPersonWidget" native="true">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="prvNameLabel">
        <property name="text">
         <string>ImiÄ™:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="prvNameLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="prvSurnameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="prvSurnameLabel">
        <property name="text">
         <string>Nazwisko:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="prvPESELLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="prvPESELLabel">
        <property name="text">
         <string>PESEL:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="prvAddressLineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="prvAdressLabel">
        <property name="text">
         <string>Adres:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="clientDialogButtonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clientEditOkButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clientEditCancelButton">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
